tests/roles/dataplane_adoption/defaults/main.yaml

```diff
@@ -64,7 +64,7 @@ default_timesync_ntp_servers:
 edpm_node_hostname: standalone.localdomain
 edpm_user: root
 edpm_nodes:
-  standalone:
+  "{{ edpm_node_hostname | default('standalone') }}":
```

Comment:
this didn't work like that it was written literally :/

https://logserver.rdoproject.org/74/51274/15/check/data-plane-adoption-osp17-multinode-to-extracted-crc/d537a42/controller/ci-framework-data/logs/openstack-k8s-operators-openstack-must-gather/namespaces/openstack/crs/openstackdataplanenodesets.dataplane.openstack.org/openstack.yaml

```
  nodes:
    '{{ edpm_node_hostname | default(''standalone'') }}':
      ansible:
        ansibleHost: 192.168.122.106
        ansibleUser: ""
      hostName: np0004610601.localdomain
```

Reply:
i am going to fully override edpm_nodes passing it into vars.yaml with v17 @ https://review.rdoproject.org/r/c/rdo-jobs/+/52452/17/playbooks/data_plane_adoption/deploy_tripleo_run_repo_tests.yaml#102